src/main/java/TaskList.java

```diff
@@ -0,0 +1,105 @@
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class TaskList implements Serializable {
+
+    // Attributes
+    private final List<Task> taskList;
+
+    // Constructor
+
+    /**
+     * Creates a new TaskList.
+     */
+    public TaskList() {
+        this.taskList = new ArrayList<>();
+    }
+
+    public TaskList(List<Task> listOfTasks) {
+        this.taskList = listOfTasks;
+    }
+
+    // Methods
+
+    /**
+     * Adds a new task into the TaskList.
+     * @param newTask task to be added into the TaskList.
+     * @return String denoting result of adding the task into the TaskList.
+     */
+    public String createTask(Task newTask) {
```

Comment:
I like how clear the javadoc comments are. However, perhaps you could add an empty line between the description and parameter section. I noticed this similar minor issue in several other places as well.